### DISCUSSION TITLE
What is the use of bitmap-filter plug-in?

### DISCUSSION THREAD

**Discussion by Mr-LZG** [FreeRDP/FreeRDP#9775] [Q&A] [answered]
I saw the description of this plug-in as follows. What are the benefits of doing this? When do I need to do this?
`Description: this plugin deactivates and filters persistent bitmap cache.`

**Comment by akallabeth** [comment, marked as answer]
filters out [[MS-RDPEGFX] bitmap cache offer](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpegfx/890f0077-dedb-4b22-8b20-ea69b9cfcacd)

that is an optional feature allowing to cache tiles client side and reuse them on later connection attempts.
with this plugin enabled the cache offer is filtered and a (empty) response injected (to prevent the client from disconnecting)

**Comment by Mr-LZG** [comment]
ok, thanks
